controllers/datafeed_controller.py

```diff
@@ -557,22 +538,15 @@ def get(self, event_key):
 class EventDetailsGet(webapp.RequestHandler):
     """
     Fetch event details, event teams, and team details
-    FMSAPI should be trusted over FIRSTElasticSearch
     """
     def get(self, event_key):
         df = DatafeedFMSAPI('v2.0')
-        df2 = DatafeedFIRSTElasticSearch()
 
         event = Event.get_by_id(event_key)
 
         # Update event
         fmsapi_events, fmsapi_districts = df.getEventDetails(event_key)
-        elasticsearch_events = df2.getEventDetails(event)
-        updated_event = EventManipulator.mergeModels(
-            fmsapi_events,
-            elasticsearch_events)
-        if updated_event:
-            event = EventManipulator.createOrUpdate(updated_event)
+        event = EventManipulator.createOrUpdate(fmsapi_events)
```

Comment:
This bit of code needs a review 👀

Reply:
[rant] The `delistify` method is such an anti-pattern. It changes our return types in unexpected ways. We have methods named `createOrUpdate` and `mergeModels` that take *lists* but return *single objects*.

If we're merging to lists with a single object, we should return a list with a single object. If we're merging two single objects, we should return a single object. The mental overhead of keeping track that a method called `mergeModels` that takes two lists will return a single object isn't worth not indexing in to an array to get an object.